internal/character/hook/technique.go

```diff
@@ -0,0 +1,47 @@
+package hook
+
+import (
+	"github.com/simimpact/srsim/internal/global/common"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Technique = "hook-technique"
+)
+
+func (c *char) Technique(target key.TargetID, state info.ActionState) {
+	techniqueTarget := c.engine.Retarget(info.Retarget{
+		Max:          1,
+		IncludeLimbo: false,
```

Comment:
nit : you can safely remove this line as the default behavior is to exclude `onLimbo` targets.